### PR TITLE
Adds css grid to admin dashboard. 

### PR DESCRIPTION
**Pull request by holgertidemand**
#### Description
Changes proposed in this pull request:

* Add overall styling

#### Screenshots:
screen shot 2018-04-19 at 21 48 47
screen shot 2018-04-19 at 21 48 56

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -38,6 +30,60 @@ ul.nav a {
...
+  width: 215px;
+}
+
+.btn {
+  width: 165px;
+  margin: 5px;
+}
+
+
+@media (min-width: 767.98px){
```
Comment (kmthorsnes) [inline app/assets/stylesheets/application.scss:55]: "min-width: 767.98px"  ????

**Review by kmthorsnes — APPROVED** [review]
Looks good to me !

**Review by davidboll — APPROVED** [review]
Nicely done!

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (holgertidemand) [reply app/assets/stylesheets/application.scss:55]: Apparently the exact breaking point for tablets. I don't know bootstrap docs said to use that ;)

**Review by larathordar — APPROVED** [review]
Looks nice

**Pull request merged**